machines/pcx86/compaq/lte286/index.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: COMPAQ LTE/286
+permalink: /machines/pcx86/compaq/lte286/
+machines:
+  - id: compaq-lte286-640k
+    type: pcx86
+    debugger: true
+    debug: true
+    uncompiled: true
+    compiled: false
```

Comment:
I think there's just an *uncompiled* machine variable...
```suggestion
```

Reply:
> I think there's just an _uncompiled_ machine variable...

@jeffpar Sorry, those flags didnt get cleaned up.  I'll try to fix that in my next PR